chatconfig/config.json

```diff
@@ -0,0 +1,484 @@
+{
```

Comment:
@chibie this file still needs to be removed

Reply:
@thelostone-mc okay, I just removed the file.